doctr/utils/geometry.py

```diff
@@ -4,20 +4,52 @@
 # See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
 
 from typing import List
-from .common_types import BoundingBox, Polygon4P
+import numpy as np
+import cv2
+from .common_types import BoundingBox, Polygon4P, RotatedBbox
 
-__all__ = ['bbox_to_polygon', 'polygon_to_bbox', 'resolve_enclosing_bbox']
+__all__ = ['rbbox_to_polygon', 'bbox_to_polygon', 'polygon_to_bbox', 'polygon_to_rbbox',
+           'resolve_enclosing_bbox', 'resolve_enclosing_bbox', 'fit_rbbox']
 
 
 def bbox_to_polygon(bbox: BoundingBox) -> Polygon4P:
     return (bbox[0], (bbox[1][0], bbox[0][1]), (bbox[0][0], bbox[1][1]), bbox[1])
 
 
+def rbbox_to_polygon(rbbox: RotatedBbox) -> Polygon4P:
+    (x, y, w, h, alpha) = rbbox
+    alpha = alpha * np.pi / 180
+    pt1 = [x + (w / 2 * np.cos(alpha)) - (h / 2 * np.sin(alpha)), y + (w / 2 * np.sin(alpha)) + (h / 2 * np.cos(alpha))]
+    pt2 = [x - (w / 2 * np.cos(alpha)) - (h / 2 * np.sin(alpha)), y - (w / 2 * np.sin(alpha)) + (h / 2 * np.cos(alpha))]
+    pt3 = [x - (w / 2 * np.cos(alpha)) + (h / 2 * np.sin(alpha)), y - (w / 2 * np.sin(alpha)) - (h / 2 * np.cos(alpha))]
+    pt4 = [x + (w / 2 * np.cos(alpha)) + (h / 2 * np.sin(alpha)), y + (w / 2 * np.sin(alpha)) - (h / 2 * np.cos(alpha))]
+    return (pt1, pt2, pt3, pt4)
```

Comment:
Since this is only numpy, if we use it in the actual running code sequence, we should vectorize it